Sources/MapboxDirectionsCLI/ProcessCommand.swift

```diff
@@ -70,6 +97,22 @@ class ProcessCommand<ResponceType : Codable, OptionsType : DirectionsOptions > :
         }
     }
     
+    private func interpolate(route: Route?) -> [CLLocationCoordinate2D?] {
+        guard let route = route else { return [] }
```

Comment:
It doesn’t appear to be necessary for this method’s argument to be optional.

Reply:
yep thanks for pointing that out!